### DISCUSSION TITLE
I'm Qurious about the structure of Zluda

### DISCUSSION THREAD

**Discussion by SkyLeo00** [vosen/ZLUDA#233] [Q&A]
As i understood, complied cuda code is imported to hip library via using zluda. Is it right? Also, i want to check out how it will happen in actual code. Anyone can explain about this process for me roughly? 
If someone answered my question, it will be really helpful for me. Thank you.

**Comment by vosen** [comment]
ARCHITECTURE.md has a broad overfiew of how it works: https://github.com/vosen/ZLUDA/blob/master/ARCHITECTURE.md
